.gitignore

```diff
@@ -9,3 +9,6 @@ node_modules
 pm2
 dump.rdb
 *.log
+
+# Metadata file generated by OS X
+.DS_Store
```

Comment:
Fwiw, I've always been of the opinion that Mac OS users should add this to their global `.gitignore`, because it's platform-specific. (the same way e.g. VSCode users should add `.vscode` to theirs)